model/database/metrics.yaml

```diff
@@ -133,3 +134,39 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.cosmos.operation.request_charge
+    type: metric
+    metric_name: db.cosmos.operation.request_charge
+    brief: "Request Charge Consumed by the operation"
+    instrument: histogram
+    unit: "{request_units}"
+    stability: experimental
+    extends: attributes.db.cosmos.minimal
+
+  - id: metric.db.cosmos.operation.max_item_count
+    type: metric
+    metric_name: db.cosmos.operation.max_item_count
+    brief: "Max Item Count Requested by the Feed (Query, ReadAll, ReadMany, Change Feed) operation"
+    instrument: gauge
```

Comment:
Gauge does not allow min, max, avg. per interval, does it?

Reply:
From my understanding, a gauge is the appropriate instrument for recording information that is non-additive in nature, unlike counters. 
However, it only returns the most recent recorded value rather than a series of values. Prometheus offers functionality to calculate the average, minimum, and maximum over a specified time period.

Reply:
Gauge is used to measure something like a temperature - it'd record the last value among all reported in the time series over period of time. 

You probably want to report it as a histogram so you have min, max, sum and each operation participating in the aggregated value.

Reply:
Discussed with Kiran offline, decided to keep it, although we agreed that, it is not super important metrics.